velox/substrait/SubstraitToVeloxPlan.h

```diff
@@ -80,7 +80,10 @@ class SubstraitVeloxPlanConverter {
   core::PlanNodePtr toVeloxPlan(const ::substrait::SortRel& sortRel);
 
   /// Convert Substrait Plan into Velox PlanNode.
-  core::PlanNodePtr toVeloxPlan(const ::substrait::Plan& substraitPlan);
+  /// Prefix: the prefix of registered functions.
+  core::PlanNodePtr toVeloxPlan(
+      const ::substrait::Plan& substraitPlan,
+      const std::string& prefix = "");
```

Comment:
Shouldn't the prefix be passed to the constructor?

Reply:
Moved to the constructor.